kafka/src/main/java/com/opentable/logging/KafkaAppender.java

```diff
@@ -63,10 +62,10 @@ public void start()
     @Override
     public void stop()
     {
-        getStatusManager().add(new InfoStatus("About to close Kafka producer", this));
+        addInfo("About to close Kafka producer");
```

Comment:
The default `addInfo` implementation handles the context not having been set yet.  Previously, this would NPE.